### PR TITLE
Initial whitepaper import

### PR DESCRIPTION
**Pull request by hiddentao**
Initial import of whitepaper markdown into mainsite.

* Markdown -> React, including YAML greymatter and footnotes
* Images imported

Screenshot 2020-05-16 at 11 54 28

### CODE REVIEW

File: mainsite/pages/whitepaper.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import styled from '@emotion/styled'
+import { flex } from 'emotion-styled-utils'
+
+import { content, images } from '../docs/whitepaper'
+import { withApollo } from '../src/frontend/hoc'
+import Layout from '../src/frontend/components/Layout'
+import Markdown from '../src/frontend/components/Markdown'
+import FundUs from '../src/frontend/components/FundUs'
+import Seo from '../src/frontend/components/Seo'
+import ContentWrapper from '../src/frontend/components/ContentWrapper'
```
Comment (HineshMandalia) [inline mainsite/pages/whitepaper.js]: We should try and keep these warnings down to a minimum

File: mainsite/package.json [hunk 2]
```diff
@@ -12,12 +12,13 @@
     "url": "git+https://github.com/safetyscore/safetyscore.git"
   },
   "scripts": {
-    "build": "next build",
+    "prepare-docs": "./scripts/prepare-docs.js",
```
Comment (HineshMandalia) [inline mainsite/package.json]: This doesn't work on my windows machine, consider the following:
```suggestion
    "prepare-docs": ".\"/scripts/prepare-docs.js\"",
```

Comment (hiddentao) [reply mainsite/package.json]: Thanks, changed

File: mainsite/pages/whitepaper.js (hunk 1 above)
Comment (hiddentao) [reply mainsite/pages/whitepaper.js]: At some point we should go through and clean up all lint warnings.

**Review by HineshMandalia — APPROVED** [review]

**Pull request merged**